### PR TITLE
II-121 add withdraw function

### PR DESCRIPTION
**Pull request by jinyoungkim927**
Feat: 
- Added prepare withdraw (rolls up transactions)
- Added complete withdraw
Fix: 
- Correct / consistent formatting for the withdraw (using react bootstrap buttons and colour scheme)
Screen Shot 2022-08-05 at 12 12 43 pm
g

### CODE REVIEW

File: src/components/WalletFeatures/WalletFeatures.tsx [hunk 1]
```diff
@@ -1,47 +1,26 @@
...
+// import "./WalletFeatures.css"
+import { Link, ImmutableXClient, ImmutableMethodResults, ERC721TokenType, ETHTokenType, ImmutableRollupStatus  } from '@imtbl/imx-sdk';
+import { useEffect, useState } from 'react';
 import DepositFunction from "./DepositFunction";
-// import WithdrawFunction from "./WithdrawFunction";
+import WithdrawFunction from "./WithdrawFunction";
+// import BalanceFunction from "./BalanceFunction";
 
 const WalletFeatures = () => {
+  const [wallet, setWallet] = useState('undefined');
```
Comment (pano-skylakis) [inline src/components/WalletFeatures/WalletFeatures.tsx:9]: Do you need to use the wallet context here instead of useState?

**Review by pano-skylakis — COMMENTED** [review]
Any tests we can add here? 😉

**Comment by jinyoungkim927** [comment]
> Any tests we can add here? 😉

Absolutely, we're planning to implement tests next week. Unfortunately our wallet page generally doesn't have any tests.

File: src/components/WalletFeatures/WalletFeatures.tsx [hunk 2]
```diff
@@ -1,47 +1,26 @@
...
+  const link = new Link('https://link.ropsten.x.immutable.com')
+  const [client, setClient] = useState<ImmutableXClient>(Object);
   return (
     <nav className="WalletFeatures-container">
```
Comment (aerim-yi) [inline src/components/WalletFeatures/WalletFeatures.tsx:13]: if we don't use the CSS file, this className won't be required.

**Review by aerim-yi — CHANGES_REQUESTED** [review]

**Review by pano-skylakis — APPROVED** [review]

**Review by nur221282 — APPROVED** [review]

**Review by aerim-yi — APPROVED** [review]

**Pull request merged**